In reqsign-http-send-reqwest, custom_client example fails to compile

Working on Fedora Asahi Remix 42, `aarch64`, using the system `rustc` (1.89.0):

```
$ git clone https://github.com/apache/opendal-reqsign.git
$ cd opendal-reqsign/
$ cd context/http-send-reqwest/
$ cargo test
[…]
   Compiling reqsign-http-send-reqwest v1.0.0 (/home/ben/src/forks/opendal-reqsign/context/http-send-reqwest)
error[E0599]: no method named `danger_accept_invalid_certs` found for struct `ClientBuilder` in the current scope
  --> context/http-send-reqwest/examples/custom_client.rs:32:10
   |
28 |       let client = Client::builder()
   |  __________________-
29 | |         .timeout(Duration::from_secs(30))
30 | |         .pool_max_idle_per_host(10)
31 | |         .user_agent("reqsign-example/1.0")
32 | |         .danger_accept_invalid_certs(false)
   | |         -^^^^^^^^^^^^^^^^^^^^^^^^^^^ method not found in `ClientBuilder`
   | |_________|
   |

For more information about this error, try `rustc --explain E0599`.
error: could not compile `reqsign-http-send-reqwest` (example "custom_client") due to 1 previous error
```

From https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.danger_accept_invalid_certs,

> This requires the optional `default-tls`, `native-tls`, or `rustls-tls(-...)` feature to be enabled.

The following is one possible fix:

```diff
diff --git a/context/http-send-reqwest/Cargo.toml b/context/http-send-reqwest/Cargo.toml
index 3f0c726..6b50dbc 100644
--- a/context/http-send-reqwest/Cargo.toml
+++ b/context/http-send-reqwest/Cargo.toml
@@ -36,3 +36,4 @@ reqwest = { version = "0.12", default-features = false }
 
 [dev-dependencies]
 tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
+reqwest = { version = "0.12", default-features = false, features = ["default-tls"] }
```

--- a/context/http-send-reqwest/Cargo.toml
+++ b/context/http-send-reqwest/Cargo.toml
@@ -32,8 +32,8 @@ bytes.workspace = true
 http.workspace = true
 http-body-util = "0.1.2"
 reqsign-core.workspace = true
-reqwest = { version = "0.12", default-features = false }
+reqwest = { workspace = true, default-features = false }
 
 [dev-dependencies]
 tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
-reqwest = { version = "0.12", default-features = false, features = ["default-tls"] }
+reqwest = { workspace = true, default-features = false, features = ["default-tls"] }

